README.md

```diff
@@ -57,6 +57,12 @@ environment to keep the build output stable.
 
 The toolchain image build files are kept in [testdata/docker/](testdata/docker/).
 
+### Regeneration Testdata on non-x86 platforms
```

Comment:
Could this be moved to `testdata/docker/README.md` under the `Building` section?